Python/Python.sublime-syntax

```diff
@@ -3,8 +3,10 @@
 name: Python
 file_extensions:
   - py
+  - py3
   - pyw
   - pyi
+  - pt
```

Comment:
Can you provide any reference for where this extension is used and what it means?

I found `py3` in [some SO question](http://stackoverflow.com/a/18032741/1327727), but I've never actually seen it. Still, I guess that counts.